Source/monster.cpp

```diff
@@ -598,7 +599,7 @@ void ClrAllMonsters()
 		Monst->_mfuty = 0;
 		Monst->_moldx = 0;
 		Monst->_moldy = 0;
-		Monst->_mdir = random_(89, 8);
+		Monst->_mdir = (direction)random_(89, 8);

```

Comment:
```suggestion
		Monst->_mdir = static_cast<direction>(random_(89, 8));

```

Here we could also use NUM_Direction.

Reply:
it dosent exist, and cannot exist, take a look at the actual enum.